server_admin/topics/users/user-registration.adoc

```diff
@@ -1,30 +1,18 @@
-[[_user-registration]]
+[id="assembly-user-registration_{context}"]
+= User Registration
 
-=== User Registration
+[role="_abstract"]
+You can use {project_name} as a third-party authorization server to manage application users. Administrators can add users or users can register themselves.
 
-You can enable {project_name} to allow user self registration.  When enabled, the login page has a registration
-link the user can click on to create their new account.
+You can enable {project_name} to allow user self-registration. When enabled, the login page displays a registration link that the user can click to create an account.
 
-When user self registration is enabled it is possible to use the registration form to detect valid usernames and emails.
-It is also possible to enable <<_recaptcha,reCAPTCHA Support>>.
+When user self-registration is enabled, the user must complete a form to add profile information. 
```

Comment:
I think this sentence is unnecessary.  The preceding sentence about creating an account implies you will need to add profile details for the account.  At least they don't need to know now.